bin/MadGraph5_aMCatNLO/cards/ZZhad_ALP_linear/ZZhad_ALP_linear_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model ALP_linear_UFO
+define q = u c d s b u~ c~ d~ s~ b~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+generate g g > z z, z > q q, z > q q
```

Comment:
it should be enough to use "generate g g > z z, z > q q"

Reply:
Hi @menglu21, indeed, this works. I will recreate the PR. thanks!